recipes/sphinxcontrib-confluencebuilder/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sphinxcontrib-confluencebuilder" %}
+{% set version = "1.4.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sphinxcontrib-confluencebuilder-{{ version }}.tar.gz
+  sha256: 6a54b99144a8b0769bde4a75bc5d9e7dc4ebfe016584d26be60b084ab1906042
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - sphinx-build-confluence = sphinxcontrib.confluencebuilder.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+    - future >=0.16.0
+    - requests >=2.14.0
+    - sphinx >=1.8
+
+test:
+  imports:
+    - sphinxcontrib
+    - sphinxcontrib.confluencebuilder
+  commands:
+    - pip check
+    - sphinx-build-confluence --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/sphinx-contrib/confluencebuilder
+  summary: Sphinx extension to output Atlassian Confluence Storage Markup documents and publish to Confluence instances.
+  license: BSD-3-Clause
```

Comment:
```suggestion
  license: BSD-2-Clause
```

Reply:
Thanks! Looks like there may be a bug in grayskull...